demos/query.md

```
@@ -0,0 +1,231 @@
+```shell
+aicr skill --agent claude-code
+```
+
+When you are in Claude: 
+
+```shell 
+claude
+```
+
+> What driver version would deploy in EKS with H100 in Ubuntu for training?
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)